documentation/src/main/markdown/gettingstarted/terms/tools.md

```diff
@@ -1,12 +1,21 @@
-# [solution_name] tools
+# Tools
 
-[solution_name] tools are run to enable the scanning of your code.
+[solution_name] tools are components that enable the scanning of your source code. [solution_name] uses several underlying tools to perform scanning including:   
+* Detector (for inspecting package manager dependencies)   
+* Signature Scanner (for inspecting the file system)   
+* Docker Inspector (for inspecting Docker container content)   
+* Bazel detector ( to discover dependencies in Bazel projects)   
+* Binary Analysis (used to determine components within binary files)   
+* Vulnerability Impact Analysis Tool (generates a Vulnerability Impact Analysis Report)   
+* IaC Scanner (supports infrastructure as code scanning)   
 
-The default tools that are run are:
+Optional properties can be specified to explicitly enable or disable these underlying tools, but by default [solution_name] will run both the Detector and Signature Scanner on the code being analyzed.
```

Comment:
All tools are executed but only those with criteria of the tool will run its scan. For example, signature scans often run because it only requires a source path, which by default is the current working directory.

Reply:
From a users' standpoint, if the tool code executes, but there is no output because it was not configured or there was nothing provided to scan, would we still say it was "run"?
Perhaps I can reword the bit about Detector and Signature Scanner to note that they run as long as there is a path provided with something to scan?

Reply:
Yes, please reword.